### DISCUSSION TITLE
HTML Report Dark and Light Mode

### DISCUSSION THREAD

**Discussion by CalebCyrill** [badeball/cypress-cucumber-preprocessor#1216] [Q&A] [answered]
Are there any options for the HTML report to enable light mode? I see in the documentation that enabled and output available

**Comment by badeball** [comment]
You need to be more specific in your tickets and link to whatever you’re referencing.

Where are you reading about light / dark mode?

**Reply by CalebCyrill** [reply]
I am asking if there are more options to the HTML report than just enable and output as shown below (referencing https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/html-report.md)
```
{
  "html": {
    "enabled": true
    "output": output path
  }
}
```

The reason why I ask is because I noticed that the report is generated based off of the system preferences, rather than being able to toggle it on the page or set the theme of the report to light or dark.

**Reply by badeball** [reply, marked as answer]
There are no other options aside from the documented ones. The HTML report (which is a small react application) honors operating system preferences, using [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme). This is determined by the browser, which means you can change your OS settings and open the same report with different results, or use EG. chrome devtools to emulate another preference.
